tests/test_emulators.py

```diff
@@ -128,27 +177,31 @@ def test_routing(self):
         #########
 
         # In this test the HRU IDs range from 0 to 2, and the subbasin IDs are 10, 20
-        model.rvc.basin_state.index = 10
+        model.rvc.basin_state.index = 10  # <<<<<<< THIS NEEDS TO BE TWO INSTANCES
+        model.rvc.basin_state.index = 20  # <<<<<<< THIS NEEDS TO BE TWO INSTANCES
```

Comment:
ToDo. We need to create two of these blocks in the RVC file

Reply:
done